docs/pages/setup/reference/cli.mdx

```diff
@@ -1289,6 +1289,68 @@ Print cluster version:
 tctl version
 ```
 
+## Resource Filtering
+
+Both `tsh` and `tctl` allow you to filter node, app, db, and kube resources using the `--search` and `--query` flags.
```

Comment:
```suggestion
Both `tsh` and `tctl` allow you to filter `node`, `app`, `db`, and `kube` resources using the `--search` and `--query` flags.
```
An alternative would be to say "filter Nodes, applications, databases, and Kubernetes resources"